shell/artifacts/Restaurants/source/RestaurantList.js

```diff
@@ -6,98 +6,109 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
-defineParticle(({DomParticle}) => {
+defineParticle(({DomParticle, html}) => {
 
-  let item = `
-<div class="item">
-  <div class="info">
-    <div class="title"><img class="icon" src="{{icon}}"><span>{{name}}</span></div>
-    <div class="address">{{address}}</div>
-    <div class="rating-outer">
-      <div class="rating" style="{{ratingStyle}}"></div>
+  const item = html`
+
+<div item style%="{{image}}" style="position: relative;">
+  <div scrim></div>
+  <div style="flex: 1"></div>
+  <div address>{{address}}</div>
+  <div style="display: flex; position: relative;">
+    <div title style="flex: 1;">{{name}}</div>
```

Comment:
I bet this changed the structure just enough to break the restaurant list test. The current selector looks like `arc-host,div[slotid="root"],div.item,div.title`, but now probably needs to be something like `arc-host,div[slotid="root"],div[item],div[title]`?

That should be an easy fix either as part of this cl or we can take a look tomorrow.

Reply:
Maybe we can attach an ID to this node instead of using a structural query? It should be less brittle and as a testing convention it can make it clear when a node is relevant to tests.

Reply:
Yes! That'll let us remove some of the intermediate structure from the query as well (which I was using to disambiguate).